notifications-kibana-plugin/common/index.ts

```diff
@@ -0,0 +1,2 @@
+export const PLUGIN_ID = 'opendistroNotificationKibana';
+export const PLUGIN_NAME = 'opendistroNotificationKibana';
```

Comment:
should it be `opendistroNotificationsKibana`? to be consistent with notification**s** plugin. also missing license headers on source files

Reply:
good catch, I'll fix the naming in this PR. For the license header. I'll post another PR to update along with other documentation, legal/liscense files